src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -92,6 +93,9 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
             return true;
         } else if (GraphIteratorSavedModel::is_supported(model_path)) {
             return true;
+        } else if (GraphIteratorProtoTxt::is_supported(model_path)) {
```

Comment:
Reduce complexity and keep same logic as your already used, check ends_with(".pbtxt") first. As I can see from code above (for binary protobuf) we doesn't expect "any file which might be interpreted as a text protobuf".

Reply:
Agreed with @slyalin, we want to be unbound from file extension and to check it without this because we want to have MO with minimal options so that user can just specify only path to the model.